src/tests/ftest/util/ior_utils.py

```diff
@@ -135,18 +132,15 @@ def run_ior(client_file, ior_flags, iteration, block_size, transfer_size,
     attach_info_path = basepath + "/install/tmp"
     try:
 
-        ior_cmd = (
-            orterun_bin + " -N {} --hostfile {} -x DAOS_SINGLETON_CLI=1 "
-            "-x CRT_ATTACH_INFO_PATH={} ior {} -s {} -i {} -a DAOS -o {} -b {} "
-            "-t {} -- --daos.pool {} --daos.svcl {} --daos.recordSize {} "
-            "--daos.stripeSize {} --daos.stripeCount {} --daos.aios {} "
-            "--daos.objectClass {} ".format(slots, client_file,
-                                            attach_info_path, ior_flags,
-                                            seg_count, iteration, filename,
-                                            block_size, transfer_size,
-                                            pool_uuid, svc_list, record_size,
-                                            stripe_size, stripe_count, async_io,
-                                            object_class))
+        ior_cmd = orterun_bin + " -np {} --hostfile {} --map-by node " \
```

Comment:
Could you enhance the get_ior_cmd() so that it creates the ior_daos_cmd or ior_mpiio_cmd.   The method would return an IOR cmdline or orterun IOR cmdline.   Additional inputs to get_ior_cmd() would be (mode => doas or mpiio) and (wrapper =>  orterun or None)    Then you would only need a single run_ior method to execute the cmdline.       I am working on soak and need to enhance get_ior_cmd() to generate the IOR cmdlines for -daos and -mpiio so I could make those changes at another time if you need to get this change in now.

Reply:
I can do that as part of a separate patch. Doing that will impact all the IOR tests present. Hence this should be dealt as a separate ticket and patch. I will create the ticket and push the patch.